src/CoreBundle/ResourceCollection/ResourceCollectionInformationService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\ResourceCollection;
+
+use ChameleonSystem\CoreBundle\Interfaces\ResourceCollectorInterface;
+
+class ResourceCollectionInformationService implements ResourceCollectionInformationServiceInterface
```

Comment:
This is (currently) specific to "locking", so -> ResourceCollectionLockingService (or similar)?

Reply:
It is not only related to locking - it indicates if the resource collection is enabled and is collecting resources.